README.install.md

```diff
@@ -63,6 +63,21 @@ Alternatively, install Nmstate system-wide:
 ```shell
 pip uninstall -y nmstate; pip install .
 ```
+### Install nmstate from tarball
+
+Download the tarball and the signature file. Then, verify the signature:
+```bash
+gpg2 --recv-keys F7910D93CA83D77348595C0E899014C0463C12BB
+gpg --verify nmstate-<version>.tar.gz.asc nmstate-<version>.tar.gz
+```
+
+Extract the tarball and install nmstate.
+
+```bash
+tar xzvf nmstate-<version>.tar.gz
+python nmstate-<version>/setup.py build
```

Comment:
I guess this should be `python3`, also in the next line

Reply:
Done!